pkgs/by-name/hu/hullcaster/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hullcaster";
+  version = "v0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "gilcu3";
+    repo = pname;
+    rev = version;
+    hash = "sha256-TaELX/xMxm7OTmVnvkgEmdhnVrIlxSNqlE73+I5qxCc=";
+  };
+
+  cargoHash = "sha256-FeIZu/9yEk8U4a1AhqHyJBhpTP453km33FemwfhZckc=";
+
+  # workaround to error: Could not create filepath: /homeless-shelter/.local/share
+  doCheck = false;
```

Comment:
I didn't check in detail, but can we just disable the offending test, and keep the others?
You can look for something like cargoTestFlags or something like that in nixpkgs to see how to do that.

Reply:
thanks, i changed it to skip just the bad test now